ouroboros-network/src/Ouroboros/Network/PeerSelection/LedgerPeers.hs

```diff
@@ -63,10 +70,21 @@ data IsLedgerPeer = IsLedgerPeer | IsNotLedgerPeer
 
 newtype NumberOfPeers = NumberOfPeers Word16 deriving Show
 
-newtype LedgerPeersConsensusInterface m = LedgerPeersConsensusInterface {
-      lpGetPeers :: SlotNo -> STM m (Maybe [(PoolStake, NonEmpty RelayAccessPoint)])
+data LedgerStateJudgement = YoungEnough | TooOld
+  deriving (Eq, Show)
+
+-- | Return ledger state information and ledger peers.
+--
+data LedgerPeersConsensusInterface m = LedgerPeersConsensusInterface {
```

Comment:
Things that we share with `consensus` should live in `ouroboros-network-api`.

Reply:
Right!